sw/otbn/code-snippets/randomness.s

```diff
@@ -111,11 +110,38 @@ test_urnd:
   /* Read the URND WSR again. */
   bn.wsrr w1, 0x2 /* URND */
 
-  /* Fail the test if the two numbers read from URND are equal. */
+  jal x0, error_checking
```

Comment:
Should also still end in a `ret`.

Reply:
Same as [here](https://github.com/lowRISC/opentitan/pull/10783#discussion_r805877085).